packages/shopify-codemod/test/transforms/remove-return-voids.js

```diff
@@ -0,0 +1,14 @@
+import 'test-helper';
+import removeReturnVoids from 'remove-return-voids';
+
+describe('removeReturnVoids', () => {
+  it('Removes void returns from else generated by Decaf', () => {
```

Comment:
Minor quibble, but lowercase first word (so that it seems like a sentence with `it`)